File: app/src/main/java/com/chess/cryptobot/model/response/livecoin/LivecoinAddressResponse.java
package com.chess.cryptobot.model.response.livecoin;

import com.chess.cryptobot.model.response.AddressResponse;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LivecoinAddressResponse extends LivecoinResponse implements AddressResponse {

    @SerializedName("wallet")
    @Expose
    private String wallet;

    @Override
    public String getAddress() {
        return wallet;
    }
}

File: app/src/main/java/com/chess/cryptobot/model/response/livecoin/LivecoinAddressResponse.kt
package com.chess.cryptobot.model.response.livecoin

import com.chess.cryptobot.model.response.AddressResponse
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LivecoinAddressResponse : PoloniexResponse(), AddressResponse {
    @SerializedName("wallet")
    @Expose
    override val address: String? = null
}